File: server/todolistJava01/src/main/java/jp/co/linnefromice/todolist01/model/Task.java
package jp.co.linnefromice.todolist01.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;
import java.util.Date;

@Entity
@Table(name = "task")
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Task {

    @Id
    private Integer id;

    private String user_id;

    private String task_title;

    private String task_description;

    private Boolean completed;

    private Date registered_date;

}

File: server/todolistKotlin01/src/main/kotlin/jp/co/linnefromice/todolistKotlin01/model/Task.kt
package jp.co.linnefromice.todolistKotlin01.model

import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table
import java.util.Date

@Entity
@Table
class Task {

    @Id
    val id : String = String()
    val user_id : String = String()
    val task_title : String = String()
    val task_description : String = String()
    val completed : Boolean = false
    val registered_date : Date? = null
}